client/client.go

```diff
@@ -43,7 +43,7 @@ type Client interface {
 	SendOutboundMessage(ctx context.Context, nodeName string, peerID string, op uint32, msgBody []byte) (*rpcpb.SendOutboundMessageResponse, error)
 	Close() error
 	SaveSnapshot(ctx context.Context, snapshotName string) (*rpcpb.SaveSnapshotResponse, error)
-	LoadSnapshot(ctx context.Context, snapshotName string) (*rpcpb.LoadSnapshotResponse, error)
+	LoadSnapshot(ctx context.Context, snapshotName string, rootDataDir string) (*rpcpb.LoadSnapshotResponse, error)
```

Comment:
I shall prefer to pass rootDataDir as optional argument, as Start() does (same as execPath and pluginDir)

Reply:
See for that the updated code for LoadSnapshot, at current main

Reply:
fixed